docs/Supported-Platforms.md

```diff
@@ -128,8 +128,8 @@ strongly about trying to get Brim running on a [non-recommended platform](#non-r
 ## Development Tools
 
 There are two primary development tools on which Brim depends:
-[Electron](https://www.electronjs.org/docs/tutorial/support#supported-platforms)
-and [Go](https://golang.org/doc/install#requirements). Their support
+[Electron](https://www.electronjs.org/docs/latest/development/README)
```

Comment:
Interestingly, Electron's content formerly at that "supported platforms" link is not only gone, but I can't for the life of me find a new page anywhere that divulges the same information. Their redirect page _claims_ that README link is the new place, but I don't see the detail there or anywhere below that. @jameskerr looked to and he can't find anything either. So I'm just pointing where Electron suggested and letting it be vague. 🤷